test/pkg/ptphelper/ptphelper.go

```diff
@@ -54,7 +54,7 @@ func GetProfileLogID(ptpConfigName string, label *string, nodeName *string) (id
 		if err != nil {
 			return id, fmt.Errorf("could not get any profile line, err=%s", err)
 		}
-		return matches[len(matches)-1][logIDIndex], nil
+		return strings.ReplaceAll(matches[len(matches)-1][logIDIndex], "{level}", ""), nil
```

Comment:
add a comment:
// 4.16+ adds log levels in message_tag: [ptp4l.0.config:{level}]